src/main/java/mod/steamnsteel/tileentity/structure/LargeFanTE.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2014 Rosie Alexander and Scott Killen.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 3 of the License, or (at your option) any
+ * later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, see <http://www.gnu.org/licenses>.
+ */
+package mod.steamnsteel.tileentity.structure;
+
+import com.foudroyantfactotum.tool.structure.registry.StructureDefinition;
+import net.minecraft.util.EnumFacing;
+
+public class LargeFanTE extends StructureTemplate
```

Comment:
Structure Template is used more so as means to stop me writing out boilerplate code.

The original use of IFluidHandler haunts its way back into the Structure system, under IStructureFluidHandler. The intention (within Structure) was to have an identical base to work from so others wouldn't go all custom on it.